.idea/navEditor.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes I think it was generate, but it cause me trouble when i want to change my working branch, so that's why i've pushed it. I have to learn more about gitHub, sometimes it cause trouble that i don't really understand.

Reply:
Manage to remove this file from the repo please.

Reply:
Add to gitignore